utils/swift_build_support/swift_build_support/products/earlyswiftdriver.py

```diff
@@ -41,6 +41,9 @@ def is_before_build_script_impl_product(cls):
         return True
 
     def should_build(self, host_target):
+        if self.is_cross_compile_target(host_target):
```

Comment:
In a follow on commit, can you add a comment here? Also we should ping @artemcm and @nkcsgexi so they know about this.